src/main/scala/nl/knaw/dans/bag/v0/DansV0Bag.scala

```diff
@@ -186,7 +186,7 @@ class DansV0Bag private(private[v0] val locBag: LocBag) extends DansBag {
       .map(_.asScala)
       .collect {
         case Seq(userId) => userId
-        case userIds if userIds.size > 1 => throw new IllegalStateException(s"Only one EASY-User-Account allowed; found ${ userIds.size }")
+        case userIds if userIds.size > 1 => throw new IllegalStateException(s"Only one EASY-User-Account allowed; found ${userIds.size}")
```

Comment:
not sure if this change is intended... my formatter went the other way around yesterday.